spec/features/user_can_signup_spec.rb

```diff
@@ -18,7 +19,7 @@
     scenario 'it has a name field' do
```

Comment:
Don't think this test is achieving anything. Would be good to remove at some point

Reply:
Noted!